vmdb/app/models/configured_system_foreman.rb

```diff
@@ -6,6 +6,22 @@ class ConfiguredSystemForeman < ConfiguredSystem
   belongs_to :customization_script_medium
   belongs_to :customization_script_ptable
 
+  def configuration_location_name
+    configuration_location.name
+  end
```

Comment:
@Fryguy @matthewd @tenderlove Is there a better way to do this?  It feels like it is making the file noisy. Maybe:
```ruby
define_method(:configuration_location_name) { configuration_location.name }
```